d2/src/main/java/com/linkedin/d2/balancer/strategies/degrader/DegraderTrackerClientUpdater.java

```diff
@@ -1,7 +1,6 @@
 package com.linkedin.d2.balancer.strategies.degrader;
 
 import com.linkedin.d2.balancer.clients.DegraderTrackerClient;
-import com.linkedin.d2.balancer.clients.TrackerClient;
```

Comment:
Any thoughts to get rid of this class completely?

Reply:
This is the old code, in the new implementation we will not use it. In that sense, we will get rid of it once we finish migration. For now, to keep the minimal change and minimal risk for the refactor change (which is not guarded by any client config change), I'd rather keep this class.

Reply:
Agree with Ruxin, see above comment.